README.md

```diff
@@ -24,7 +24,9 @@ This project is a reboot of Kafdrop 2.x, dragged kicking and screaming into the
 # Requirements
 
 * Java 11 or newer
-* Kafka (version 0.10.0 or newer)
+* Kafka (version 0.11.0 or newer)
+
+> Note: DO NOT offer backwards compatibility for older version of Kafka.
```

Comment:
I think that line is superfluous; we don't need it.

Reply:
@ekoutanov 
OK, removed.